packages/transactions/src/PaymentV2.ts

```diff
@@ -51,10 +51,9 @@ export default class PaymentV2 extends Transaction {
     this.signature = opts.signature
   }
 
-  serialize(): Uint8Array {
+  serialize(forSigning: boolean = false): Uint8Array {
```

Comment:
Probably need to update the other serialize methods to make multisig signing easier, if this pattern is desirable

Reply:
I think instead of passing a boolean lets pass an object with optional params:

You can make this line:
` serialize({ includeSignature = false }): Uint8Array {`

And then in `/transactions/src/Transaction.ts` make line 16:
` abstract serialize({ includeSignature } : { includeSignature?: boolean }): Uint8Array`

Reply:
@tyler-whitman - I can do that but the provided code isn't an optional param. When I implement as suggested, I have to pass an empty "options object" to the method. (`{}`). Just wanted to ask for you to clarify so we can align on the preferred method. Thanks!

Reply:
Yeah thats a good point, and we cant assign default values in an abstract function. I guess we would have to do something like:

`abstract serialize(includeSignature?: boolean): Uint8Array`

Reply:
I ended up completely removing the parameter. Not sure why I had it present in the first place, to be honest 🤔